data/datasets.py

```diff
@@ -750,27 +750,27 @@ def __init__(self, ijbc_data_root):
         self.match = pd.read_csv(op.join(test1_dir, 'match.csv'))
 
         self.transforms = transforms.Compose([
-            transforms.Normalize([.5, .5, .5], [.5, .5, .5]),
             transforms.Resize([112, 112]),
-            transforms.toTensor()
+            transforms.ToTensor(),
+            transforms.Normalize([.5, .5, .5], [.5, .5, .5]),
```

Comment:
Maybe this is the reason why the performance degrades

Reply:
Why?